packages/engine/paima-sm/src/types.ts

```diff
@@ -262,6 +288,20 @@ export type ChainDataExtensionCardanoDelegation = ChainDataExtensionBase &
     cdeType: ChainDataExtensionType.CardanoPool;
   };
 
+export const ChainDataExtensionCardanoProjectedNFTConfig = Type.Object({
+  type: Type.Literal(CdeEntryTypeName.CardanoProjectedNFT),
+  contract: Type.String(),
+  scheduledPrefix: Type.String(),
+  startSlot: Type.Number(),
+  stopSlot: Type.Optional(Type.Number()),
+  name: Type.String(),
```

Comment:
I wonder if we should allow specifying a list of NFT collections you want to filter too. It seems like maybe over-optimizing, but it may cut down on the size of the Carp requests so it may actually make a difference (since if many contracts are using projected NFTs, filtering Carp queries for a specific set of collections may reduce the number of times you have to paginate)

Not a high priority for v1 I guess

Reply:
we can do it easily, but i'd change `asset` to `policy id` and `asset name` for better search options

Reply:
https://github.com/dcSpark/carp/pull/167